docs/start_stop.md

```diff
@@ -0,0 +1,89 @@
+# Advanced start, stop and status
+
+## Starting
+
+### All services
+
+To spin up entire monaca-docker (including both **log** and **metric**
+pipelines) one must execute:
+
+```sh
+$ docker-compose -f log-pipeline.yml -f docker-compose.yml up
+```
+
+from the root of the [monasca-docker][1] repository.
+
+Invidual pipelines can still be launched. See below for more details
+
+#### Metric pipeline
+
+For **metric** pipeline it suffices to just execute
+
+```sh
+$ docker-compose up
+```
+
+or
+
+```sh
+$ docker-compose -f docker-compose.yml up
+```
+
+There is 3rd approach which requires passing all (some) of the members of
+**metric** pipeline as arguments after ```up```. For example:
+
+```sh
+$ docker-compose up kafka zookeeper monasca-notification
+```
+
+It will result in launching only **kafka**, **zookeeper** and **monasca-notification**.
+
+#### Log pipeline
+
+The **log** pipeline is a bit more complicated. As it is an addition to the
+**metric** pipeline, some of the services are
+shared between each and thus defined in [docker-compose.yml](../docker-compose.yml) without a duplicate
+in **log-pipeline.yml**. That effects the way you could launch *just* the **log** pipeline. See the
+examples below to get a grip:
+
+* both pipelines ```docker-compose -f log-pipeline.yml -f docker-compose.yml up```
+* just **log** pipeline ```docker-compose -f log-pipeline.yml -f docker-compose.yml up {{
+  log_pipeline_members }}``` where ```log_pipeline_members``` resolves to all the services defined in
+  [log-pipeline.yml](../log-pipeline.yml) file + some of the services that are
+  shared between i.e **kafka**, **zookeeper**, **keystone**, **mysql**. Obviously all the *-init*
+  containers corresponding to aforementioned services should be included in launch as well.
+
+### Invidual services
+
+Launching individual services has been described above to some extent. In greater detail, launching an
+individual service is as easy as specifying what service(s) you wish to launch as an argument to the
+```docker-compose up``` command. For example, to launch **monasca-log-transformer** you execute
+something like ```docker-compose -f log-pipeline up log-transformer```. The tricky point is to known
+the "dependencies" of single service. These "dependencies" need to be available, either externally or
+launched together with the desired service.
+
+## Stopping
+
+Everything stays the same, as for the launching. The difference is that instead of doing
```

Comment:
same **** as

Reply:
Changed the first sentence to *Stopping the services it similar to starting them*